setenv.FNX_SUBMITTER_APP.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
   <key>Label</key>
-  <string>my.startup</string>
+  <string>technology.fnx</string>
```

Comment:
why not `fnx.technologies`?  that's what I'd intuitively look for if I were to debug anything related to this.

Reply:
Following convention, see: https://en.wikipedia.org/wiki/Reverse_domain_name_notation

<img width="619" alt="image" src="https://user-images.githubusercontent.com/11287980/98610604-cc26ae00-22a4-11eb-8ea0-e73438e65b49.png">